### DISCUSSION TITLE
Time Editor / Picker?

### DISCUSSION THREAD

**Discussion by axelcasauran** [serenity-is/Serenity#6289] [Help] [answered]
Hi,

Is there a time editor or picker?
If there is a sample in the demo --I would appreciate if you can direct me to the link.

**Comment by dfaruque** [comment, marked as answer]
`[TimeEditor]` is not working?

**Reply by axelcasauran** [reply]
got it thanks..

**Reply by gfo2007** [reply]
@axelcasauran is the time editor working for you? I get the error in the image when trying to using it to save a record that contains time.

![Time Editor Error](https://user-images.githubusercontent.com/14317420/199980267-819637b9-75ac-472d-8751-64cf6893fd1f.PNG)

**Reply by sapster77** [reply]
I get the same error as you. Also the saved time from the record is not shown. When I try to input a new value, i get the error shown above. If I use DateTimeEditor instead of TimeEdfitor everything works correctly. I just dont want the date....
Did you ever find a fix for this?

**Comment by VictorTomaili** [comment]
TimeEditor works with integer fields. It converts the hours and minutes to minutes as integer.

**Reply by sapster77** [reply]
Ah Ok. Thanks for that - i mistakenly thought it extracted time from DateTime. 
Thanks again:-)
